tests/attested_tls/sgxlkl_enclave/alpine/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright Microsoft. 
+# Licensed under the attached Microsoft Software License Terms
```

Comment:
This can be removed if we compile the application in `sgxlkl_enclave/Dockerfile`, as #664  did.